src/shared/components/person/settings.tsx

```diff
@@ -45,11 +51,17 @@ import { SortSelect } from "../common/sort-select";
 import Tabs from "../common/tabs";
 import { CommunityLink } from "../community/community-link";
 import { PersonListing } from "./person-listing";
+import { InitialFetchRequest } from "shared/interfaces";
```

Comment:
Even tho this compiles, I'm like 80% sure this will fail. The build system does not liken these style imports.